decidim-core/app/packs/src/decidim/data_picker.js

```diff
@@ -235,10 +237,8 @@ export default class DataPicker {
   _removeValue($picker, target) {
     if (target) {
       this._setCurrentPicker($picker, target);
-      this.current.target.fadeOut(500, () => {
```

Comment:
💬 I'm not sure if removing the fadeout is a good idea. My concern is that it can be confusing for the participant not to see the selected option to disappear. Did you try to add the `:slow` metadata to the test? If that doesn't work, maybe a slepp(500) can do the trick?